convert2rhel/unit_tests/systeminfo_test.py

```diff
@@ -68,9 +78,33 @@ def tearDown(self):
         if os.path.exists(unit_tests.TMP_DIR):
             shutil.rmtree(unit_tests.TMP_DIR)
 
+    @unit_tests.mock(tool_opts, "no_rpm_va", True)
+    def test_generate_modified_rpms_with_no_rpm_va(self):
+        self.assertEqual(system_info.generate_modified_rpms(), None)
+
+    @unit_tests.mock(tool_opts, "no_rpm_va", False)
+    @unit_tests.mock(logger, "LOG_DIR", unit_tests.TMP_DIR)
+    @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked(("rpm1\nrpm2\n", 0)))
+    @unit_tests.mock(utils, "get_file_content", GetFileContentMocked(filename='rpm_va.log', data=['rpm1', 'rpm2']))
+    def test_generate_modified_rpms_without_difference(self):
+        self.assertEqual(system_info.generate_modified_rpms(), None)
```

Comment:
Likewise as above, we don't know if this returns `None` because of `no_rpm_va` being implemented incorrectly or it returns the differences